src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -705,29 +683,15 @@ export class ActivityFillGraphQuerier {
     if (enableCPUUsage && threadCPUDelta) {
       // It must be non-null because we are checking this in the processing
       // step and eliminating all the null values.
-      const cpuDeltaBefore = ensureExists(threadCPUDelta[sample]);
+      cpuBeforeSample = ensureExists(threadCPUDelta[sample]);
```

Comment:
I think I preferred the name `cpuDeltaBefore` because it makes it clear that it's a delta. Or maybe `cpuDeltaBeforeSample` would be even better. The same applies to the parameter of the `_accumulateInBuffer` function.